keycloak.jenkins.io.tf

```
@@ -0,0 +1,33 @@
+resource "postgresql_database" "keycloak" {
+  name  = "keycloak"
+  owner = postgresql_role.keycloak.name
+}
+
+resource "random_password" "pgsql_keycloak_user_password" {
+  length           = 24
+  override_special = "!#%&*()-_=+[]{}:?"
+  special          = true
+}
+
+resource "postgresql_role" "keycloak" {
+  name     = "keycloak"
+  login    = true
+  password = random_password.pgsql_keycloak_user_password.result
+}
```

```suggestion
resource "random_password" "pgsql_keycloak_user_password" {
  length           = 24
  override_special = "!#%&*()-_=+[]{}:?"
  special          = true
}

resource "postgresql_role" "keycloak" {
  name     = "keycloak"
  login    = true
  password = random_password.pgsql_keycloak_user_password.result
}

resource "postgresql_database" "keycloak" {
  name  = "keycloak"
  owner = postgresql_role.keycloak.name
}
```
(nit) easier with the declaration of element first then their use in another element.